jsx/Form.js

```diff
@@ -1299,6 +1299,8 @@ class ButtonElement extends React.Component {
       <div className="row form-group">
         <div className={this.props.columnSize}>
           <button
+            {...(this.props.id ? {id: this.props.id} : {})}
+            {...(this.props.name ? {name: this.props.name} : {})}
```

Comment:
I'm unsure why the spread operator would be necessary here.

Reply:
@HenriRabalais The spread operator is handling both the key and the property (ex. id='btnExample). The id key or name key may not have values and so they shouldn't be included in rendering or it would be id='' & name='' inside the button component. I believe it's necessary, thoughts?

Reply:
From what I can see, the ternary operator is accomplishing that goal by checking the values exist, and then setting the the key-value pair if they do, and an empty object otherwise. Where does the spread operator come into play here? The spread operator is reserved for iterables, such as an array. It doesn't seem like either of these ternary operations result in an expression that needs to iterated over.